### DISCUSSION TITLE
Why is dukun not working?

### DISCUSSION THREAD

**Discussion by Ixchinia** [thatskymod/Sky-CotL-Scripts#90] [General]
It always crashes when we allow certain nodes like the unlock all emotes/closet and it can't even blink teleport nor auto CR? Can you devs fix this?

**Comment by thatskymod** [comment]
### Update finished
All files from the [Libs/](https://github.com/thatskymod/Sky-CotL-Scripts/tree/main/Canvas/Libs) folder have now been replaced by their newer counterparts, including Dukun Langit.

Thank you for standing by while the issues were being addressed.
